CHANGELOG.md

```diff
@@ -2,6 +2,95 @@
 
 ## 28
 
+### Call Caching
```

Comment:
Beefy meaning too much ? There's more extensive doc in the README so I could just link to that